### ISSUE TITLE
Error loading the PCB into KiCad

### ISSUE THREAD

**Issue by Taki-B** [solderparty/pcb_holder_ruler#1]
Hi!
When I try to load any of the PCB I an error warning that prevents opening the file. For example for the 25 mm, 1.6 mm ruler I get:

![Screenshot_2022-02-12_15-20-51](https://user-images.githubusercontent.com/19894039/153715424-f363479f-06e1-4302-b438-147e6d14bf9f.png)

Line 118 looks like this:

  (stroke (width 0) (type solid)) (fill solid) (layer "F.SilkS") (tstamp a576bd65-0612-4f7a-afee-d84c8007d5c8))

I'm using KiCad 6.0.1 and I'm an absolute beginner. Can it be that there's a date code pointing to the 32nd of December as the error message suggests?

Kind regards,

Taki

**Comment by arturo182** [comment]
Hmm, this is probably because these were made using v6.99 aka. nightly. 
What you could try is to open the PCB file in a text editor and change the date in the first line by hand, you can see it here: https://github.com/solderparty/pcb_holder_ruler/blob/main/jig_25mm_1.6mm.kicad_pcb#L1 and then try to open it again in v6.0.1

**Comment by Taki-B** [comment]
Thanks for the reply Arturo! I replaced the version/date as you suggested into 20211014, which I just copied from a 6.0.1 PCB file, but I now get the next error, which seems to point to a different file format that the nightly uses:

![Screenshot_2022-02-13_00-33-03](https://user-images.githubusercontent.com/19894039/153732235-032fcf71-06b1-4bc7-a841-d4ccc0e0b821.png)

As I don't see any nightly built for Linux, I'll have to wait for an official update before trying again to open these PCB files. You might consider adding an information, which version is needed to open the files in the README, though.
